src/ert/simulator/simulation_context.py

```diff
@@ -68,12 +68,15 @@ def _run_forward_model(
     ):
         queue_evaluators = [
             partial(
-                job_queue.stop_long_running_jobs,
+                job_queue.stop_long_running_realizations,
                 ert.ert_config.analysis_config.minimum_required_realizations,
             )
         ]
 
-    asyncio.run(job_queue.execute(evaluators=queue_evaluators))
+    try:
+        asyncio.run(job_queue.execute(evaluators=queue_evaluators))  # type: ignore
+    except Exception:
```

Comment:
bare exceptions are not a good idea.  Being bare, it should at least be re-raised?

Reply:
There is a bare exception catch in queue.py already. Should not be needed.

Reply:
Yes, but the point is that this gets raised in here only.

Reply:
I think the queue should be responsible for catching the exception(s) here yes.